CHANGELOG.md

```diff
@@ -71,6 +71,11 @@ Additions to existing modules
   ++⁺ˡ : Reflexive R → ∀ zs → (_++ zs) Preserves (Pointwise R) ⟶ (Pointwise R)
   ```
 
+* In `Data.Maybe.Properties`:
+  ```agda
+  maybe′-∘ : ∀ {b} (f : B → C) (g : A → B) → f ∘ (maybe′ g b) ≗ maybe′ (f ∘ g) (f b)
```

Comment:
Sorry to miss this earlier: we generally suppress leading quantifier prefixes where they can (easily ;-)) be inferred directly, or by reference to the actual code, so
```suggestion
  maybe′-∘ : ∀ f g → f ∘ (maybe′ g b) ≗ maybe′ (f ∘ g) (f b)
```
should suffice!